helm/botkube/values.yaml

```diff
@@ -505,3 +505,18 @@ extraObjects: []
 #      - kind: ServiceAccount
 #        name: "{{ include \"botkube.serviceAccountName\" . }}"
 #        namespace: "{{ .Release.Namespace }}"
+
+## Parameters for the test container with E2E tests.
+e2eTest:
+  image:
+    registry: docker.io
+    repository: pkosiec/botkube-test # TODO: Replace with image built on `develop`
+    tag: latest
```

Comment:
I will replace this in a follow-up PR after merging this PR and building test image.

Reply:
Yes, you're totally right 👍 I will change that before merge then